### PR TITLE
[DOCFIX] Update docs with new component labeling guidelines

### PR DESCRIPTION
**Pull request by aaudiber**
Apologies in advance for deleting trailing whitespace, but I don't think we want trailing whitespace in our markdown files, and many editors remove trailing whitespace by default.

### CODE REVIEW

**Comment by aaudiber** [comment]
alluxio-bot, check this please

File: docs/en/Contributing-to-Alluxio.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ For a tutorial, see the GitHub guides on
 [ALLUXIO-100] Implement an awesome new feature
 ~~~~~
 
+-   Add a GitHub label describing which component of Alluxio your change falls under. If you're
```
Comment (gpang) [inline docs/en/Contributing-to-Alluxio.md]: Are there some guidelines of what the component names cover?

**Comment by gpang** [comment]
@aaudiber I left a minor comment. Thanks!

**Comment by aaudiber** [comment]
@gpang I've added that components correspond with root-level directories

**Comment by RongGu** [comment]
LGTM.

**Comment by gpang** [comment]
LGTM

**Pull request merged**